cromshell

```diff
@@ -194,7 +194,7 @@ function checkPrerequisites
 function assertCanCommunicateWithServer
 {
   # Make sure we can talk to the cromwell server:
-  serverName=$( echo ${1}| sed 's#.*://##g' )
+  serverName=$( echo ${1}| sed 's#.*://##g'| sed 's#:.*##g' )
```

Comment:
It would be good to make this more restrictive with anchors and numerals only.

Can you make the sed statement `'s#:[0-9]*$##g'`